ebuild-maintenance/general/text.xml

```diff
@@ -0,0 +1,111 @@
+<?xml version="1.0"?>
+<guide self="ebuild-maintenance/general/">
+<chapter>
+<title>Maintaining Your Package</title>
+
+<section>
+<title>General responsibilities</title>
+<body>
+
+<note>
+Gentoo's Quality Assurance (QA) team also maintains a
+<uri link="https://projects.gentoo.org/qa/policy-guide/">policy guide</uri>,
+which contains useful advice on ebuild structure and behaviour.
+</note>
+  
+<p>
+You should always use the latest <uri link="::ebuild-writing/eapi/">EAPI</uri>.
+
+The main exception is for critical packages like Portage or
+those typically maintained by base-system which normally wait
+~2-3 years.
+
+Note that for new EAPIs, we require 1 year before using them in
+stable ebuilds.
+
+Another common exception is when an eclass you consume does not yet support it.
+You should file a bug or otherwise contact the eclass maintainer to enquire
+about its status with respect to the new EAPI.
+
+<e>Rationale: Newer EAPIs add improvements which are often user-visible,
+e.g. EAPI 6's eapply_user and EAPI 7's improved cross-compile support.</e>
+</p>
+
+<p>
+Check regularly for new bugs on
+<uri link="https://bugs.gentoo.org/">bugs.gentoo.org</uri>.
+</p>
```

Comment:
"Respond and resolve bugs assigned to you. If you do not use package assigned to you anymore or can't maintain it, pass it on to someone else." (with a link to https://devmanual.gentoo.org/general-concepts/package-maintainers/index.html#adding-and-removing-maintainers)